commons-numbers-arrays/src/main/java/org/apache/commons/numbers/arrays/ExtendedPrecision.java

```diff
@@ -204,6 +204,28 @@ private static double productLowUnscaled(double x, double y, double xy) {
         return productLow(hx, lx, hy, ly, xy);
     }
 
+    /**
+     * Compute the low part of the double length number {@code (z,zz)} for the exact
+     * square of {@code x} using Dekker's mult12 algorithm. The standard precision product
+     * {@code x*x} must be provided. The number {@code x} is split into high and low parts
+     * using Dekker's algorithm.
+     *
+     * <p>Warning: This method does not perform scaling in Dekker's split and large
+     * finite numbers can create NaN results.
+     *
+     * @param x Number to square
+     * @param xx Standard precision product {@code x*x)
+     * @return the low part of the square double length number
+     * @see #productLowUnscaled(double, double, double)
```

Comment:
Javadoc issue: The productLowUnscaled is private. So you cannot refer to it in a package private method.